### PR TITLE
Move code in saito_controller into main.rs #36

### PR DESCRIPTION
**Pull request by tharindal**
Moved code from the saito_controller.rs file into the main.rs and removed the saito_controller.rs file. Thread creation broken into separate methods.

Signed-off-by: Tharinda Lunuwila <tharinda.gl@gmail.com>

### CODE REVIEW

File: saito-rust/src/main.rs [hunk 1]
```diff
@@ -45,7 +319,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     tracing_subscriber::fmt::init();
 
     let configs = Arc::new(RwLock::new(
-        ConfigHandler::load_configs("configs/saito.config.json".to_string())
+        ConfigHandler::load_configs("configs/saito.config.template.json".to_string())
```
Comment (SankaD) [inline saito-rust/src/main.rs:322]: revert this change. template is to be copied into saito.config.json file

Comment (tharindal) [reply saito-rust/src/main.rs:322]: Okay

**Review by SankaD — APPROVED** [review]
